pkg/logrusutil/logrusutil.go

```diff
@@ -47,10 +45,6 @@ func PropagateJSON(logger *logrus.Logger, jsonLine []byte, header string, begin
 	case logrus.TraceLevel:
 		logger.Trace(header + j.Msg)
 	}
-	return
-
-fallback:
-	logrus.Info(header + string(jsonLine))
 }
```

Comment:
Not necessary to remove goto

Reply:
I removed the `goto` because there are no other instances of `goto` in Lima's codebase.

Reply:
I agree with @AkihiroSuda that removing a `goto` doesn't make sense if the price for it is duplicating code.

I guess the linter doesn't support this, but my personal view is: `goto` is fine as long as you jump "forward", and only out of control structures, i.e. it is not used to create a loop. With that restriction it just become a fancy version of `break`.

Reply:
Returned back `goto`.